### PR TITLE
feat: align swap k-line and same-chain token selection (OK-57650, OK-57751)

### PR DESCRIPTION
**Pull request by zhaono1**
[OK-57650](https://onekeyhq.atlassian.net/browse/OK-57650) [OK-57751](https://onekeyhq.atlassian.net/browse/OK-57751)

----------

## Summary

- Enable the existing native chart-controls mode for the Swap K-line view.
- Add a `Same chain` network shortcut to the Swap token selector based on the opposite token's network.
- Keep the shortcut first in compact and full selectors, preserve disabled bridge-only states, recalculate `N+`, and align the new UI with the Figma dimensions.

## Intent & Context

OK-57650 aligns the Swap K-line controls with the Market native control layer without changing chart data or trading behavior.

OK-57751 makes same-chain selection immediately visible in the combined Swap & Bridge token selector. When selecting the From token, the To token network is featured; when selecting the To token, the From token network is featured. This behavior is limited to the regular Swap selector.

## Root Cause

- The Swap `TradingViewV2` adapter did not opt in to `enableNativeChartControls`, so the hosted chart rendered its legacy control UI.
- The token selector rendered fixed slices of the network list and had no presentation model for a featured network. It could not pin the opposite token network without duplication, keep a disabled featured network visible, or calculate `N+` from the rendered list.

## Design Decisions

- Enable the existing chart-control opt-in only at the ordinary Swap K-line adapter and preserve its data source, storage namespace, callbacks, and disabled features.
- Derive `sameChainNetwork` as selector presentation data instead of mutating the underlying network order.
- Keep Limit and Stock selectors unchanged.
- Pass optional featured-network metadata through the generic chain selector without adding Swap dependencies to shared components.
- Keep disabled non-featured networks filtered while allowing the disabled featured network to remain visible.
- Use the exact Figma logical dimensions for the 36px compact controls, 1x24 divider, 48px featured row, and 9px full-list divider region.

## Changes Detail

- `SwapKLineContent.tsx`: enable native chart controls on the existing `TradingViewV2` instance and hide the duplicate selector.
- Extend the network filter item with optional badge, responsive image-size, and text props.
- Add and test a Swap-only network view model that extracts the opposite token network and removes its duplicate.
- Update the compact network row and full chain selector with featured-network ordering, badge, divider, disabled state, and responsive `N+` behavior.
- Update generated translations in a dedicated commit.

## Issues

- Fixes OK-57650
- Fixes OK-57751

## Risk Assessment

- **Risk Level**: Medium
- **Affected Platforms**: Mobile / Desktop / Web / Extension
- **Risk Areas**: Hosted chart bridge compatibility, Swap From/To direction mapping, bridge-only disabled networks, and responsive compact-row counts.

## Test plan

- [x] `yarn agent:check --profile commit`
- [x] `yarn jest packages/kit/src/views/Swap/pages/modal/SwapTokenSelectModal.utils.test.ts --runInBand` (13 tests)
- [x] Verify the mobile compact row, complete `28+` label, featured full-list row, and divider in the iOS simulator.
- [ ] Verify the Swap K-line interval selector and chart updates on mobile, desktop, web, and extension.
- [ ] Verify the 640px desktop compact row and `N+` navigation at runtime.
- [ ] Verify Swap From/To direction changes and bridge-only disabled states on mobile and desktop.

[OK-57650]: https://onekeyhq.atlassian.net/browse/OK-57650
[OK-57751]: https://onekeyhq.atlassian.net/browse/OK-57751

### CODE REVIEW

File: packages/kit/src/views/Swap/pages/modal/SwapKLineContent.tsx [hunk 1]
```diff
@@ -1211,6 +1211,7 @@ function SwapKLineContentBody({
         decimal={selectedToken?.decimals ?? 0}
         dataSource="polling"
         disabledFeatures={disabledTradingViewFeatures}
+        enableNativeChartControls
```
[the remark this answers is not in the document]
Comment (zhaono1) [reply packages/kit/src/views/Swap/pages/modal/SwapKLineContent.tsx:1214]: Fixed in `bafad1c294`. `TIMEFRAME_SELECTOR` is now included in the desktop disabled-feature set, and mobile inherits the same set. Enabling native chart controls therefore no longer leaves the hosted interval selector visible. `yarn agent:check --profile commit` passed.

File: packages/shared/src/locale/json/zh_TW.json [hunk 2]
```diff
@@ -4148,6 +4150,7 @@
   "trade.privacy_mode": "隱私模式",
   "trade.privacy_mode_tooltips": "隱私模式可隱藏兌換或跨鏈時，發送與接收位址之間的鏈上關聯。",
   "trade.taking_longer_than_usual": "交易處理時間較長，建議到鏈上查看實際狀態",
+  "trade.token_same_chain": "相鏈",
```
Comment (originalix) [inline packages/shared/src/locale/json/zh_TW.json]: 启用“同链”推荐项后，这个值会直接显示在网络 Badge 中，但 zh_TW 和 zh_HK 当前都写成了语义错误的“相鏈”；同一 key 在 fr_FR 还是“Même chain”，de 仍为英文，也存在未完整本地化的问题。

这会让相关语言用户看到错误或混合语言的核心功能标签。请在 Lokalise 翻译源中修正这些 locale（繁体中文可使用“同鏈”），再通过仓库的 i18n pull 流程重新生成文件，不要直接手工维护生成的 JSON。

**Review by originalix — APPROVED** [review]

File: packages/shared/src/locale/json/de.json [hunk 3]
```diff
@@ -4148,6 +4172,7 @@
   "trade.privacy_mode": "Privatsphärenmodus",
   "trade.privacy_mode_tooltips": "Der Privatsphärenmodus kann die direkte On-Chain-Verbindung zwischen Absender- und Empfängeradressen während Swaps oder Bridges verbergen.",
   "trade.taking_longer_than_usual": "Dauert länger als gewöhnlich, überprüfen Sie den On-Chain-Status zur Bestätigung",
+  "trade.token_same_chain": "Same chain",
```
Comment (originalix) [inline packages/shared/src/locale/json/de.json:4175]: 当德语或法语用户打开普通 Swap 选币器并出现同链推荐时，这个 Badge 会分别显示英文 `Same chain` 和中英混合的 `Même chain`，因为对应 locale 尚未完成本地化。

这会让新功能的核心标签直接出现语言回退。请在 Lokalise 中补全 `trade::token_same_chain` 的 `de` 和 `fr_FR` 翻译，再通过 i18n pull 流程重新生成 locale 文件。
